### PR TITLE
AAP-19392 Add new postgres_max_connections variable for controller v4.5

### PR DESCRIPTION
**Pull request by ariordan-redhat**
AAP-19392 Add new postgres_max_connections variable for controller v4.5

Affects `/titles/aap-installation-guide`

### CODE REVIEW

**Comment by ariordan-redhat** [comment]
@AlanCoding 
This adds the variable that you created in #488

**Review by ianf77 — APPROVED** [review]
LGTM

**Review by AlanCoding — APPROVED** [review]

File: downstream/modules/platform/ref-controller-variables.adoc [hunk 1]
```diff
@@ -104,8 +104,12 @@ Default = `awx`.
...
 | *`postgres_use_ssl`* | Use this variable if postgreSQL uses SSL.
-// The postgres_max_connections variable is for Controller v4.5: Do not include it until v4.5 is released.
-// | *`postgres_max_connections`* | Max database connections setting to apply if you are using installer-managed postgreSQL. See the administration guide for help selecting a value.
+| *`postgres_max_connections`* | Maximum database connections setting to apply if you are using installer-managed postgreSQL.
+
+See link:{BaseURL}/red_hat_ansible_automation_platform/{PlatformVers}/html-single/automation_controller_administration_guide/index#ref-controller-database-settings[PostgreSQL database configuration] in the {ControllerName} administration guide for help selecting a value.
+
+Default for RPM installations = 200 for a single node 
+and 1024 for a cluster.
 | *`receptor_listener_port`* | Port to use for recptor connection.
```
Comment (ianf77) [inline downstream/modules/platform/ref-controller-variables.adoc:113]: receptor

**Review by ianf77 — COMMENTED** [review]
Looks OK to me.

There's a change in the line after.

**Pull request merged**